UPGRADE-2.1.md

```
@@ -6,3 +6,6 @@
 
 1. The `sylius_admin.dashboard.index.content.latest_statistics.new_customers` hook has been deprecated and disabled. 
    It has been replaced by the `sylius_admin.dashboard.index.content.latest_statistics.pending_actions`.
+
+### Twig hooks
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)